SwiftUI/ReviewWQuestion-Calculator/View/ContentView.swift

```diff
@@ -0,0 +1,207 @@
+        GridItem(.flexible())
+    ]
+    
+    var body: some View {
+        VStack {
+            HStack {
+//                Text(result(stack: stack))
+//                    .font(.system(size: 90))
+//                    .minimumScaleFactor(0.5) // 폰트 크기가 절반까지 자동으로 줄어듬
+//                    .lineLimit(1) // 한줄로 출력
+                if calcResult != "오류" {
+                    Text(result(value: calcResult))
+                        .font(.system(size: 90))
+                        .minimumScaleFactor(0.5) // 폰트 크기가 절반까지 자동으로 줄어듬
+                        .lineLimit(1) // 한줄로 출력
+                } else {
+                    Text("오류")
+                        .font(.system(size: 90))
+                        .minimumScaleFactor(0.5) // 폰트 크기가 절반까지 자동으로 줄어듬
+                        .lineLimit(1) // 한줄로 출력
+                }
+
+            }
+            .padding(.init(top: 0, leading: 20, bottom: 0, trailing: 20 ))
+            .frame(width: UIScreen.main.bounds.width,height: UIScreen.main.bounds.height*0.3, alignment: .bottomTrailing)
+            
+            LazyVGrid(columns: columns, spacing: 10){
+                ForEach(0..<20) { i in
+                    Button(action: {
+                        if i == 0 {
+                            stack.choiceOperator(value: data[i].text)
+                            calcResult = "0"
+                        } else if i == 19 {
+                            if !stack.temp.isEmpty {
+                                stack.push(stack.temp)
+                            }
+                            calcResult = postfix(arr: stack.elements)
+                            stack.temp = ""
+                        } else {
+                            stack.choiceOperator(value: data[i].text)
+                            calcResult = stack.temp
+                        }
+                    }, label: {
+                        CircleButtonView(backgroundColor: data[i].backgroundColor, text: data[i].text, textColor: data[i].textColor)
+                    })
+                }
+            }
+            .frame(width: UIScreen.main.bounds.width,height: UIScreen.main.bounds.height*0.6)
+            Spacer()
+        } // VStack`
+        .preferredColorScheme(.dark)
+        .padding(.init(top: 0, leading: 10, bottom: 0, trailing: 10 ))
+    } // body
+} // ContentView
+
+
+
+func result(value: String) -> String {
+    let numberFormatter = NumberFormatter()
+    numberFormatter.maximumFractionDigits = 2 // 소수점 2번째 자리까지 출력
+    numberFormatter.numberStyle = .decimal // 3자리 마다 콤마
+    
+    return numberFormatter.string(for: Double(value)) ?? "00"
+//    return stack.elements.map{$0}.joined(separator: "").isEmpty ? "0" : numberFormatter.string(for: Double(stack.elements.map{String($0)}.joined(separator: ""))!) ?? "0"
+}
+
+func postfix(arr: [String]) -> String{
+    var numStack = Stack()
+    var opStack = Stack()
+    print("arr :  \(arr)")
+    for i in 0..<arr.count {
+        switch arr[i] {
```

Comment:
```suggestion
    for element in arr {
        switch elements {
```

다른 방식의 for-in 구문을 활용해보는 것을 혹시 생각해보셨을까요?
인덱스로 배열에 접근할 때는 `out of range` 런타임 에러가 발생할 수 있어서 안전하지 않은 방법이라는 생각이 들어요 🤔

Reply:
> 

좋은 방법인것 같습니다!